projects/1.19/assets/simple-emerald-tools-fabric/emerald_tools/lang/zh_cn.json

```
@@ -0,0 +1,152 @@
+  "item.emerald_tools.copper_shears": "铜剪刀",
+  "item.emerald_tools.copper_excavator": "铜开掘铲",
+  "item.emerald_tools.copper_hammer": "铜锤",
+  "item.emerald_tools.copper_paxel": "铜万用工具",
+  "item.emerald_tools.copper_treeaxe": "铜伐木斧",
+
+  "item.emerald_tools.wooden_excavator": "木开掘铲",
+  "item.emerald_tools.wooden_hammer": "木锤",
+  "item.emerald_tools.wooden_paxel": "木万用工具",
+  "item.emerald_tools.wooden_treeaxe": "木伐木斧",
+
+  "item.emerald_tools.gold_excavator": "金开掘铲",
+  "item.emerald_tools.gold_hammer": "金锤",
+  "item.emerald_tools.gold_paxel": "金万用工具",
+  "item.emerald_tools.gold_treeaxe": "金伐木斧",
+  "item.emerald_tools.gold_bow": "金弓",
+
+  "item.emerald_tools.iron_excavator": "铁开掘铲",
+  "item.emerald_tools.iron_hammer": "铁锤",
+  "item.emerald_tools.iron_paxel": "铁万用工具",
+  "item.emerald_tools.iron_treeaxe": "铁伐木斧",
+  "item.emerald_tools.iron_bow": "铁弓",
+
+  "item.emerald_tools.diamond_excavator": "钻石开掘铲",
+  "item.emerald_tools.diamond_hammer": "钻石锤",
+  "item.emerald_tools.diamond_paxel": "钻石万用工具",
+  "item.emerald_tools.diamond_treeaxe": "钻石伐木斧",
+  "item.emerald_tools.diamond_bow": "钻石弓",
+
+  "item.emerald_tools.netherite_excavator": "下界合金开掘铲",
+  "item.emerald_tools.netherite_hammer": "下界合金锤",
+  "item.emerald_tools.netherite_paxel": "下界合金万用工具",
+  "item.emerald_tools.netherite_treeaxe": "下界合金伐木斧",
+  "item.emerald_tools.netherite_bow": "下界合金弓",
+
+  "item.emerald_tools.emerald_helmet": "绿宝石头盔",
+  "item.emerald_tools.emerald_chestplate": "绿宝石胸甲",
+  "item.emerald_tools.emerald_leggings": "绿宝石护腿",
+  "item.emerald_tools.emerald_boots": "绿宝石靴",
+    
+  "item.emerald_tools.ruby_helmet": "红宝石头盔",
+  "item.emerald_tools.ruby_chestplate": "红宝石胸甲",
+  "item.emerald_tools.ruby_leggings": "红宝石护腿",
+  "item.emerald_tools.ruby_boots": "红宝石靴",
+
+  "item.emerald_tools.amethyst_helmet": "紫水晶头盔",
+  "item.emerald_tools.amethyst_chestplate": "紫水晶胸甲",
+  "item.emerald_tools.amethyst_leggings": "紫水晶护腿",
+  "item.emerald_tools.amethyst_boots": "紫水晶靴",
+    
+  "item.emerald_tools.steel_helmet": "钢头盔",
+  "item.emerald_tools.steel_chestplate": "钢胸甲",
+  "item.emerald_tools.steel_leggings": "钢护腿",
+  "item.emerald_tools.steel_boots": "钢靴",
+  
+  "item.emerald_tools.obsidian_helmet": "黑曜石头盔",
+  "item.emerald_tools.obsidian_chestplate": "黑曜石胸甲",
+  "item.emerald_tools.obsidian_leggings": "黑曜石护腿",
+  "item.emerald_tools.obsidian_boots": "黑曜石靴",
+  
+  "item.emerald_tools.copper_helmet": "铜头盔",
+  "item.emerald_tools.copper_chestplate": "铜胸甲",
+  "item.emerald_tools.copper_leggings": "铜护腿",
+  "item.emerald_tools.copper_boots": "铜靴",
+
+  "item.emerald_tools.emerald_arrow": "绿宝石箭",
+  "item.emerald_tools.ruby_arrow": "红宝石箭",
+  "item.emerald_tools.amethyst_arrow": "紫水晶箭",
+  "item.emerald_tools.diamond_arrow": "钻石箭",
+
+  "item.emerald_tools.arrow.pierce": "穿透+%s",
+  "item.emerald_tools.arrow.punch": "冲击+%s",
+  "item.emerald_tools.arrow.damage": "伤害+%s",
+  
+  "itemGroup.emerald_tools.emerald_tools_group" : "简单宝石工具"
```

```suggestion
  "itemGroup.emerald_tools.emerald_tools_group" : "简单工具"
```
不知道另一个pr的模组名称是否应该修改成这样

---

我翻译时主要参考mcmod上的介绍，但也一直拿不定标题，感觉还是这样好点（这下forge、1.18fabric、1.19fabric标题翻译各不相同了...

---

应该也行吧（）
等一手其他审核

---

都不太行，and more 怎么说

---

`简单工具`肯定不行，`Emerald`也不见了，毕竟也是模组特点之一
个人意见是叫`简单宝石工具与物品`，但是可能太长了（

---

这里后来有讨论结果吗（）

---

似乎没有（）

---

首先 Emerald 是绿宝石，完整翻译应为`易用绿宝石工具和其他物品`

---

群里有一个`更多拓展及绿宝石工具`（由迷尘提出）
感觉这个前后换下，改成`绿宝石工具及更多拓展`或者`简单/简易绿宝石工具及更多拓展`会好些
~~不知道会不会太长~~

---

`绿宝石工具及更多拓展`还是长啊，但是就这样吧？~~已经不想思考了~~

---

个人觉得OK的